modules/private-clusters-default.adoc

```diff
@@ -3,27 +3,25 @@
 // * installing/installing_aws/installing-aws-government-region.adoc
 // * installing/installing_aws/installing-aws-private.adoc
 // * installing/installing_gcp/installing-gcp-private.adoc
+// * installing/installing_azure/installing-azure-government-region.adoc
 // * installing/installing_azure/installing-azure-private.adoc
 
 [id="private-clusters-default_{context}"]
 = Private clusters
 
 ifeval::["{context}" == "installing-aws-government-region"]
-:gov:
+:aws-gov:
 endif::[]
 
-ifndef::gov[]
-If your environment does not require an external internet connection, you can deploy a private {product-title} cluster that does not expose external endpoints. Private clusters are accessible from only an internal network and are not visible to the Internet.
-endif::gov[]
-ifdef::gov[]
 You can deploy a private {product-title} cluster that does not expose external endpoints. Private clusters are accessible from only an internal network and are not visible to the Internet.
 
+ifdef::aws-gov[]
 [NOTE]
```

Comment:
This note isn't showing up for me, so please double check before merging.

Reply:
This is actually meant for the AWS Gov docs; I had to do a tweak in this PR to separate it from Azure Gov, since it doesn't apply here :slightly_smiling_face: . It shows up correctly in the AWS section. Thanks!